### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by mstrutt**
First real punt at unit test, would really appreciate some feedback if you don't mind :)

### CODE REVIEW

File: tests/unit/broswe-storage-service.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		it('should set and retrieve a string', function() {
+			browserStorage.setItem('foo', 'bar');
+
+			expect(browserStorage.getItem('foo')).toEqual('bar');
+		});
+
+		it('should allow a specific value to be removed', function() {
+			browserStorage.setItem('foo', 'bar');
+			browserStorage.removeItem('foo');
+			expect(browserStorage.getItem('foo')).not.toEqual('bar');
```
Comment (brainsiq) [inline tests/unit/broswe-storage-service.js]: I would change your assertion to be a bit more specific to what you actually expect to be returned once foo is removed as I could change the removeItem function to set the value to 'somethingelse' and it'd still pass this test.

File: tests/unit/broswe-storage-service.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		});
+
+		it('should allow all values to be removed', function() {
+			browserStorage.setItem('foo', 'bar');
+			browserStorage.setItem('bar', 'baz');
+
+			expect(browserStorage.getItem('foo')).toEqual('bar');
+			expect(browserStorage.getItem('bar')).toEqual('baz');
```
Comment (brainsiq) [inline tests/unit/broswe-storage-service.js]: Take out these 2 additional assertions that are covered by the first test?

File: tests/unit/session-service.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+					id: 'my id'
+				}
+			});
+		});
+
+		it('should subsequently exist', function() {
+			expect(session.exists()).toBeTruthy();
+		});
+
+		it('should return the info used when created', function() {
```
Comment (brainsiq) [inline tests/unit/session-service.js]: This one test looks the same as the test that a valid session should return info, except in one you are testing your API (create) and in the other looks like you are testing the implementation (storing this in browser session from what I can tell?). Generally would avoid doing that later case as it ties your tests to your implementation code and causes headaches when someone wants to change it,

File: tests/unit/broswe-storage-service.js (hunk 1 above)
Comment (mstrutt) [inline tests/unit/broswe-storage-service.js]: nice spot, I hadn't considered that

**Comment by mstrutt** [comment]
Thanks for the feedback :) I've updated that tests accordingly

File: app/index.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
 	<body layout="column">
 		<md-toolbar layout="row">
 			<div class="md-toolbar-tools">
-				<h2>Personalisation Portal</h2>
+				<h2>Ufified Console</h2>
```
Comment (toboid) [inline app/index.html]: little typo here

File: tests/unit/broswe-storage-service.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	var browserStorage;
+
+	beforeEach(module('ucApp'));
+
+	beforeEach(inject(function(_browserStorage_) {
+		// The injector unwraps the underscores (_) from around the parameter names when matching
+		browserStorage = _browserStorage_;
+	}));
+
+	describe('browserStorage', function() {
```
Comment (toboid) [inline tests/unit/broswe-storage-service.js:11]: Like the file. I don't use _should_ any more as I find that `it('sets and retrieves a string');` reads just as well without the boiler plate. That's just personal preference though.

File: tests/unit/session-service.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+		it('should accept credentials', function() {
+			session.storeInfo({
+				token: 'my token',
+				user: {
+					id: 'my id'
+				}
+			});
+		});
+
+		it('should subsequently exist', function() {
```
Comment (toboid) [inline tests/unit/session-service.js]: Tests shouldn't have any interdependencies and some test runners will randomise the order of the tests to identify dependencies. This test is dependent on the one above. Can probably uses a `beforeEach` hook to set up some state that is required for both tests

File: app/index.html (hunk 4 above)
Comment (mstrutt) [inline app/index.html]: _facepalm_

**Pull request merged**